core/src/main/scala/org/apache/spark/sql/delta/actions/actions.scala

```diff
@@ -448,6 +448,25 @@ case class RemoveFile(
 }
 // scalastyle:on
 
+case class DeleteFile(
```

Comment:
Why do you need a new Action? Can't we use the existing action `RemoveFile`?

Reply:
`RemoveFile` represents an ineffective file in the current snapshot, but still exists in FileSystem/Cloud Storage.
But `DeleteFile` represents an action that delete a file from FileSystem/Cloud Storage. I think there are different.